pkg/hintrunner/operand.go

```diff
@@ -133,6 +116,23 @@ func (imm Immediate) Resolve(vm *VM.VirtualMachine) (*memory.MemoryValue, error)
 	return memory.MemoryValueFromFieldElement(felt), nil
 }
 
+type Operator uint8
+
+const (
+	Add Operator = iota
+	Mul
+)
+
+type ResOperander interface {
+	Resolve(vm *VM.VirtualMachine) (*memory.MemoryValue, error)
+}
```

Comment:
I think perhaps `ResOperander` should go on top of Deref, since all subsequent type will implement this interface

Reply:
Oop, force-pushed without the change. I can fix this in another PR.